configure.ac

```diff
@@ -32,6 +32,9 @@ AC_SUBST([MRN_PACKAGE_STRING])
 MRN_BUNDLED=FALSE
 AC_SUBST([MRN_BUNDLED])
 
+MRN_DATA_DIR="${datadir}/${PACKAGE}"
+AC_SUBST([MRN_DATA_DIR])
```

Comment:
@komainu8  In the github actions from this build ( https://github.com/mroonga/mroonga/runs/8012897950?check_suite_focus=true#step:9:165 ), we can see this warning:

```
config.status: creating data/install.sql
config.status: WARNING: data/install.sql contains a reference to the variable `datarootdir'
which seems to be undefined.  Please make sure it is defined
```

I believe the resulting `data/ install.sql` must contain something like this

```
source ${datarootdir}/mroonga/install_plugin.sql
source ${datarootdir}/mroonga/update.sql
```

isn't it ?

Actually, before I noticed that there was already this work, I tried to update to mroonga 12.06 and noticed that the fix for https://github.com/mroonga/mroonga/issues/509 was not functional when using `configure` and `make`. Basically, I observed what was also reported in https://github.com/mroonga/mroonga/issues/520 .

So I tried a bit myself. I started with the same change as this one, using `AC_SUBST` in configure and noticed that problem, so I studied a bit more autoconf / automake - but I don't have much experience with these tools and they are extremely complex, so before you read I have to warn that I might be wrong 😉 . Here's what I found:

According to https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.71/html_node/Installation-Directory-Variables.html , we can not simply use `AC_SUBST` to let configure substitute `MRN_DATA_DIR`, because it is an installation directory variable and the substitution needs to be done with make ( this is the [*you should not use these variables except in makefiles*](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.71/html_node/Installation-Directory-Variables.html#:~:text=you%20should%20not%20use%20these%20variables%20except%20in%20makefiles) part). It seems it is so that a different directory can be selected during make install. According to my understanding of this documentation, the substitution should not happen during the `configure` step, it should happen during the `make` step. 

I came up with this patch https://github.com/perrinjerome/mroonga/commit/525c4a16c6a11b2474866806ebfffaedbde5ef49  and then I found out that you were already working on this, so I preferred to discuss instead of opening another pull request for the same problem. If you find it helpful to integrate this patch (or part of this patch) please go ahead, if you like I can also make a pull request with this patch. But I am a bit surprised that this was so complicated, so maybe I was doing something wrong or maybe it's just that autoconf/automake is very complex.

In any case, thank you for working on this issue.

Reply:
Thanks.
We will merge your change to this pull request.